lib/appoptics_apm/frameworks/padrino.rb

```diff
@@ -86,20 +88,18 @@ def render_with_appoptics(engine, data = nil, options = {}, locals = {}, &block)
   end
 end
 
-if defined?(::Padrino)
+if defined?(Padrino) && AppopticsAPM::Config[:padrino][:enabled]
```

Comment:
So this defaults to the global Padrino scope because there is no Padrino module declared within this file?

Reply:
We check if the global Padrino exists (loaded from the padrino gem) before we try to instrument it.